### DISCUSSION TITLE
Academic Benefits

### DISCUSSION THREAD

**Discussion by karri-venkata-kishore** [community/community#139153] [GitHub Education]
### Hi everyone,

hi 
 to community of github 
i have been verified for almost 3 days before for student pack of academic benefits still i haven't received anything would you please resolve my issue and help to sort out this one

**Comment by ferid333** [comment]
Hi,

It can take a few days for the Student Developer Pack benefits to activate after approval. If it’s been more than 3 days, I suggest checking the following:

Confirm Approval: Double-check your GitHub Education dashboard to ensure your application was fully approved.

Email Check: Make sure to check your email (including spam or junk folders) for any confirmation or instructions from GitHub Education.

Contact Support: If the issue persists, try reaching out to [GitHub Education Support](https://education.github.com/contact) for further assistance.

Hopefully, this will resolve the issue soon!

**Comment by davevad93** [comment]
Hi @karri-venkata-kishore ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @karri-venkata-kishore was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
